File: src/main/java/frc/robot/poses/java/EClimberPose.java
package frc.robot.poses.java;

public enum EClimberPose {
    Stowed(0.25),
    Intake(-0.282),
    Score(-0.05),
    L4(0.15),
    L3(0.088),
    L2(0.078),
    L1(0),
    AlgaeLow(0.15),
    AlgaeHigh(0.15),
    ClimbReady(-0.195);

    public final double Rotations;

    EClimberPose(double rotations) { Rotations = rotations; }
}

File: src/main/java/frc/robot/poses/kotlin/EClimberPose.kt
package frc.robot.poses.kotlin

/**
 * EClimberPose enum represents different positions that the
 * climber can be within the robot's system.
 *
 * @property rotation The position of the climber in rotations.
 */

enum class EClimberPose (@JvmField var rotations: Double)
{
    /** Represents the Stowed position of the climber */
    Stowed(0.0),

    /** Represents the Alignment position of the climber */
    Aligning(0.17),

    /** Represents the Climbing position of the climber */
    Climbing(-0.205),

    /** Software limits for rotation */
    ForwardLimit(0.25),

    ReverseLimit(-0.25);
}